src/Mod/TechDraw/Gui/MDIViewPage.cpp

```diff
@@ -1217,7 +1217,7 @@ bool MDIViewPage::compareSelections(std::vector<Gui::SelectionObject> treeSel, Q
 void MDIViewPage::showStatusMsg(const char* s1, const char* s2, const char* s3) const
 {
     QString msg = QString::fromUtf8("Selected: ");
-    msg.append(QObject::tr(" %1.%2.%3 ")
+    msg.append(QObject::tr("%1.%2.%3")

```

Comment:
How do I move prepend and append the whitespace to this string without including them in the string?

Reply:
Same as in SelectionSingleton::setPreselect